features/expectation_framework_integration/aggregating_failures.feature

```diff
@@ -59,7 +59,8 @@ Feature: Aggregating Failures
                 # ./spec/use_block_form_spec.rb:18
                 # ./spec/use_block_form_spec.rb:10
 
-                1.1.1) Failure/Error: expect(response.status).to eq(200)
+                1.1.1) Failure/Error:
+                         expect(response.status).to eq(200)
```

Comment:
Actually that's what I was unsure about. I made the snippets be always on the next line for consistency, but it's not a strong opinion. I'll make change to do so.

By the way there's another concern about putting the snippets on the next line. There's no problem when an RSpec's exception has a message starting with a blank line:

```
 1.1) Failure/Error:
        expect(response.status).to eq(200)

         expected: 200
              got: 404

         (compared using ==)
       # ./spec/nested_failure_aggregation_spec.rb:7
```

However it's hard to read when there's no blank line:

```
 1.2.2) Failure/Error:
          expect(response.headers).to include("Content-Length" => "21")
          expected {"Content-Type" => "text/plain"} to include {"Content-Length" => "21"}
          Diff:
          @@ -1,2 +1,2 @@
          -[{"Content-Length"=>"21"}]
          +"Content-Type" => "text/plain",
        # ./spec/nested_failure_aggregation_spec.rb:11
```

Maybe we should always put a blank line in ExceptionPresenter and remove the blank line in exception message?